olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -89,6 +89,20 @@ spec:
         x-descriptors:
         - 'urn:alm:descriptor:io.kubernetes.conditions'
       version: v1alpha1
+    - description: Represents an installation of a particular version of Knative Serving
+      displayName: Knative Serving 0.10.0 (obsolete)
```

Comment:
```suggestion
      displayName: Knative Serving (obsolete group, see documentation)
```

maybe?

Reply:
I think we need to sufficiently convey in their names why there need to be two. Ideally, we'd have some means of telling the console *NOT* to display one of our owned CRD's, which is something @bbrowning has asked for in the past.